app/presenters/simple_smart_answer_presenter.rb

```diff
@@ -17,6 +17,8 @@ def start_button_text
 
     if details["start_button_text"] == "Start now"
       I18n.t("formats.start_now")
+    elsif details["start_button_text"] == "Continue"
+      I18n.t("continue")
```

Comment:
As far as solutions go this is miserable, however this appears to be how we handle the labelling here. https://github.com/alphagov/frontend/commit/a6e9db442ada1a983ae47818075776f6c8bcf70a

We should be aware that there are potentially more hiccups we don't account for: https://github.com/alphagov/publisher/blob/21541089bd9dc07435efde92e1a6413e1fd659c7/app/views/simple_smart_answers/_fields.html.erb#L30

Reply:
Oh dear WHADAMESS. Shall we ship this now as it looks bad. But please can you create a card on the UX tech debt board: https://trello.com/b/Hamq6Ore/govuk-user-experience-tech-debt with a nice explanation of suggested fix.